### DISCUSSION TITLE
Enable Explain Execution Plan for SQL Server queries

### DISCUSSION THREAD

**Discussion by yoni-yad2** [dbeaver/dbeaver#19871] [General]
**Is your feature request related to a problem? Please describe.**
DBeaver option "Explain Execution Plan" is greyed out in SQL Server connections

**Describe the solution you'd like**
Implement option "Explain Execution Plan" by parsing output from command "SET SHOWPLAN_ALL ON"+(user SQL statement) and displaying it uniformly.

**Describe alternatives you've considered**
Using SSMS for Explain Execution Plan

**Comment by LonwoLonwo** [comment]
Hello @yoni-yad2 

Actually, we have support for the Execution Plan for the SQL Server. 

But this functionality is available in Lite-Edition, Enterprise-Edition, Ultimate-Edition and Team Editions only.

**Comment by LonwoLonwo** [comment]
You can try it for free: https://dbeaver.com/trial/

**Comment by yoni-yad2** [comment]
Seems like a basic feature to me (and you also offer it free for MySQL and PostgreSQL)
